<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNew</class>
 <widget class="QDialog" name="AddNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>407</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="login_lbl">
     <property name="text">
      <string>Введіть логін:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="login_led"/>
   </item>
   <item>
    <widget class="QLabel" name="password_lbl">
     <property name="text">
      <string>Введіть пароль:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password_led"/>
   </item>
   <item>
    <widget class="QLabel" name="secondpassword_lbl">
     <property name="text">
      <string>Підтвердіть пароль:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="secondpassword_led"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancel_pb">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_pb">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
